src/core/core-adapter.js

```diff
@@ -335,7 +335,7 @@ export default class CoreAdapter {
         this.globalStorage.delete(StorageKeys.QUERY_TRIGGER);
 
         const exposedParams = this._getOnUniversalSearchParams(
-          data[StorageKeys.UNIVERSAL_RESULTS].sections,
+          data.universalResults.sections,
```

Comment:
I'm (also presumably?) not a huge fan of using storage keys when `data` isn't a storage system but we may as well be consistent

Reply:
ah, you're right. I missed this one during the merge